trikot-viewmodels-declarative/sample/common/TRIKOT_FRAMEWORK_NAME.podspec

```diff
@@ -17,7 +17,8 @@ Pod::Spec.new do |spec|
     spec.pod_target_xcconfig = {
         'KOTLIN_BUILD_TYPE[config=Debug]' => 'DEBUG',
         'KOTLIN_BUILD_TYPE[config=Release]' => 'RELEASE',
-        'KOTLIN_TARGET[sdk=iphonesimulator*]' => 'iosX64',
+        'KOTLIN_TARGET[sdk=iphonesimulator*][arch=x86_64]' => 'iosX64',
+		'KOTLIN_TARGET[sdk=iphonesimulator*][arch=arm64]' => 'iosSimulatorArm64',
```

Comment:
re-indent here

Reply:
Seams like my newly installed text editor is using tabs instead of spaces 🤨